cmd/provenanced/config/toml.go

```diff
@@ -51,7 +51,7 @@ func WriteConfigToFile(configFilePath string, config *ClientConfig) {
 		panic(err)
 	}
 
-	if err := os.WriteFile(configFilePath, buffer.Bytes(), 0o644); err != nil {
+	if err := os.WriteFile(configFilePath, buffer.Bytes(), 0o600); err != nil {
```

Comment:
I know the linter bitches about this, but it needs to stay `0o644`. The config file needs be readable by any user.

Reply:
I changed it back with no linting, but we may want to consider making it 600. It doesn't need to be readable by any user.